WFP-VAM/prism-app
# 941 Display time-series chart data directly from map context; COUNTRY=cambodia

This fixes issue #941.

How to test the feature:

- On layers tab select a rainfall analysis
- Click on map to open tooltip, you should get list of available charts
- Click on a chart, it should display
- On the left menu if you select an other analysis, chart should update
- Into the popup when you click the download button, file should download

Screenshot/video of feature:
![image](https://github.com/WFP-VAM/prism-app/assets/100352902/c7b288a5-3a59-465f-83c0-bed8595aa0b4)
![image](https://github.com/WFP-VAM/prism-app/assets/100352902/c7b288a5-3a59-465f-83c0-bed8595aa0b4)
![image](https://github.com/WFP-VAM/prism-app/assets/100352902/0db8570b-b318-4d04-91d4-a0c8e6da1c9e)
